api/cmd/api/main.go

```diff
@@ -107,6 +113,12 @@ func main() {
 	// Endpoint to update nft backstory for the nft owner
 	r.POST("/nft/:id/backstory", handler.NewUpdateNftBackstoryHandler(nftTokenRepository))
 
+	// Endpoint to get the metadata of story nfts
+	r.GET("/nfts", handler.NewGetNftsHandler(nftTokenRepository))
+
+	// Admin Endpoint to batch update nft metadata
```

Comment:
how do you guard this admin endpoint?

Reply:
I plan to add an admin key for authentication in the next PR. Will need to integrate AWS secret manager for this